pegasus/sites.v3/code.org/public/oceans.haml

```diff
@@ -0,0 +1,48 @@
+---
+title: AI Oceans
+theme: responsive
+---
+
+:ruby
+  launch_ai_landing = DCDO.get('launch_ai_landing', nil)
+  tutorial = {
+    heading: "AI for Oceans",
+    background_image: "/images/fit-1940/sw_landingpage.jpg",
+    left_heading: I18n.t(:starwars_javascript_title),
```

Comment:
Not sure if we'll need to loc strings at all since it's an en-only tutorial for now, but I imagine you might like to use `hoc_s` when the time comes.  It's a significantly easier way to add strings!

Reply:
I chatted with @Erin007 and she confirmed the tutorial is only English for now.  Keeping i18n in mind is a good best practice, so I second your recommendation to use hoc_s.

Reply:
When I have the strings from marketing, I'll add with hoc_s. Thanks for the tip!